### PR TITLE
[Admin] CRUD TaskCriteria new feature

### LINKED ISSUE #7: [Admin] CRUD TaskCriteria
- As an admin, I want to CRUD task criteria
- Repo::TaskCriteria model:
```
Table task_criterias {
  id id
  title string required
  max_value integer required default 0
  task_id id [ref:> tasks.id]
}
```
- router `/admin/tasks/:id/criterias`

### PR DESCRIPTION
**Pull request by ochupa**
Closes #7

### CODE REVIEW

**Comment by sveredyuk** [comment]
Good job. Can you also extend `seeds.rb` with some samples?

File: config/routes.rb [hunk 1]
```diff
@@ -27,6 +27,7 @@
     end
     resources :tasks, except: %i[destroy] do
       resources :task_submissions, only: %i[index destroy], as: :submissions, path: :submissions
+      resources :criteria
```
Comment (PyrinAndrii) [inline config/routes.rb]: ??
```suggestion
      resources :criteria, except: :show
```

**Review by PyrinAndrii — COMMENTED** [review]
nice one 👍

File: config/routes.rb (hunk 1 above)
Comment (ochupa) [reply config/routes.rb]: Thanks. Added suggested changes.

**Comment by ochupa** [comment]
> Good job. Can you also extend `seeds.rb` with some samples?

Thanks @sveredyuk. I've added two criteria samples for one task.

File: db/seeds.rb [hunk 2]
```diff
@@ -56,6 +56,18 @@
...
+first_task_criterium_1 = Repo::TaskCriterium.where(
+  title: "First criterion",
+  max_value: 10,
+  task:   first_task
+).first_or_create!
+
+second_task_criterium_1 = Repo::TaskCriterium.where(
+  title: "Second criterion",
+  max_value: 5,
+  task:   first_task
```
Comment (PyrinAndrii) [inline db/seeds.rb:68]: hm, I am not sure about the business logic of it, but is it possible for tasks to have more than one criterium?
@sveredyuk could you clarify this for us?

Comment (sveredyuk) [reply db/seeds.rb:68]: Yes. task has many criteria: Code Quality, API design, Tests, etc...

**Review by sveredyuk — APPROVED** [review]

**Pull request merged**